lua/GPM.lua

```diff
@@ -619,38 +618,45 @@ metaTableGPM_ = {
 }
 
 --- Type to create a Generalised Proximity Matrix (GPM).
--- It has several strategies that can use geometry as well as Area, Intersection, Distance and Network.
--- @arg data.distance Distance around to end points (optional).
--- @arg data.destination base::CellularSpace or a Network, containing the destination points.
--- @arg data.origin A base::CellularSpace with geometry representing entry points on the network.
--- @arg data.progress print as values are being processed default is true (optional).
--- @arg data.strategy A string with the strategy to be used for creating the GPM (optional).
+-- GPM is a concept used to establish relations between geographical objects.
```

Comment:
I think that it could be explained better how GPM interacts with Network, because Network can be a destination, but, Network is not a "roads" e.g. It is a "roads" plus "ports" or "cities" e.g. Then, differently of the GPM article (Aguiar), our GPM uses three spatial objects, not just two.